hack/image/get_arch.sh

```diff
@@ -0,0 +1,6 @@
+arch=`uname -m`;
+case \
+    ${arch} in x86_64 | amd64) echo amd64 ;;\
+               ppc64el | ppc64le) echo ppc64le ;; \
+               *) echo unsupported ;; \
+esac
```

Comment:
If we just use `go env GOARCH` in Makefile, I don't think we need this file anymore?